requirements-ml.txt

```diff
@@ -1,7 +1,7 @@
 scikit-learn>=0.23.2
 keras>=2.4.3
 rapidfuzz>=2.6.1
-tensorflow-gpu>=2.6.4; sys.platform == 'linux'
+tensorflow-gpu >=2.6.4, <= 2.11; sys.platform == 'linux'
```

Comment:
I think it needs to be `<2.11`

Reply:
Not sure it does based on this: https://pypi.org/project/tensorflow-gpu/#history

Reply:
nvm, 2.11 is the correct cutoff, you're correct.

Reply:
good double check